components/admin_console/permission_schemes_settings/permission_team_scheme_settings/permission_team_scheme_settings.jsx

```diff
@@ -401,158 +401,164 @@ export default class PermissionTeamSchemeSettings extends React.Component {
+                                    />
+                                </span>
+                            </div>
                         </div>
-                        <div className='form-group'>
-                            <label
-                                className='control-label'
-                                htmlFor='scheme-description'
-                            >
-                                <FormattedMessage
-                                    id='admin.permissions.teamScheme.schemeDescriptionLabel'
-                                    defaultMessage='Scheme Description:'
-                                />
-                            </label>
-                            <textarea
-                                id='scheme-description'
-                                className='form-control'
-                                rows='5'
-                                value={schemeDescription}
-                                placeholder={localizeMessage('admin.permissions.teamScheme.schemeDescriptionPlaceholder', 'Scheme Description')}
-                                onChange={this.handleDescriptionChange}
+
+                        <AdminPanel
+                            titleId={t('admin.permissions.teamScheme.schemeDetailsTitle')}
+                            titleDefault='Scheme Details'
+                            subtitleId={t('admin.permissions.teamScheme.schemeDetailsDescription')}
+                            subtitleDefault='Set the name and description for this scheme.'
+                        >
+                            <div className='team-scheme-details'>
+                                <div className='form-group'>
+                                    <label
+                                        className='control-label'
+                                        htmlFor='scheme-name'
+                                    >
+                                        <FormattedMessage
+                                            id='admin.permissions.teamScheme.schemeNameLabel'
+                                            defaultMessage='Scheme Name:'
+                                        />
+                                    </label>
+                                    <LocalizedInput
+                                        id='scheme-name'
+                                        className='form-control'
+                                        type='text'
+                                        value={schemeName}
+                                        placeholder={{id: 'admin.permissions.teamScheme.schemeNamePlaceholder', defaultMessage: 'Scheme Name'}}
```

Comment:
Yes, you need to use the `t()` mark here `t('admin.permissions.teamScheme.schemeNamePlaceholder')`.